File: Modulo2-ObjectiveC-Lorena/Exercicio11.m
//
//  main.m
//  Exercicio11-C
//
//  Created by Lorena Rodrigues Bruno on 06/09/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

#import <Foundation/Foundation.h>

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        float salarioIntegral, salarioProporcional;
        int diasTrabalhados;
        printf("Digite o valor integral do salário: ");
        scanf("%.2f", &salarioIntegral);
        printf("Digite o número de dias trabalhados: ");
        scanf("%i", &diasTrabalhados);
        
        salarioProporcional = (salarioIntegral/22.0) * diasTrabalhados;
        printf("O salário do funcionário esse mês será de: %.2f\n", salarioProporcional);
        
    }
    return 0;
}

File: Modulo2-Lorena/exercicio11.swift
//
//  main.swift
//  Exercicio11
//
//  Created by Lorena Rodrigues Bruno on 24/08/2018.
//  Copyright © 2018 Lorena Rodrigues Bruno. All rights reserved.
//

import Foundation

print ("Digite o valor integral do salário: ")
var entradaSalario : String! = readLine()
print ("Digite o número de dias trabalhados: ")
var entradaDias : String! = readLine()

var salarioIntegral : Float! = Float(entradaSalario)
var diasTrabalhados : Float! = Float(entradaDias)

var salarioProporcional : Float = (salarioIntegral/22.0) * diasTrabalhados

print("O salário do funcionário esse mês será de: \(salarioProporcional)l")

